src/components/MovieList/MovieList.jsx

```diff
@@ -1,34 +1,49 @@
-import React, {Component} from 'react';
+import React, { Component } from 'react';
 import Axios from 'axios';
+import classNames from 'classnames';
 import MovieItem from '../MovieItem/MovieItem';
 import styles from './movie-list.scss';
 
+const cx = classNames.bind(styles);
+
 class MovieList extends Component {
+  static getMovies() {
+    return Axios.get('http://localhost:3000/api/v1/movies');
+  }
+
   constructor() {
     super();
 
     this.state = {
-      movies: []
+      movies: [],
     };
+  }
 
-    MovieList.getMovies().then(movies => {
+  componentDidMount() {
+    MovieList.getMovies().then((movies) => {
       this.setState({
-        movies: movies.data.map(movie => {
-          return <MovieItem key={movie.id} title={movie.title} description={movie.description} />;
-        })
-      })
+        movies: movies.data.map(movie => (
```

Comment:
Don't put components to the state. State must contain only 'state' of you application (data). All component should be rendered only in render method